addons/tmf_loadouts/loadouts/Norway

```diff
@@ -0,0 +1,497 @@
+	hmd[] = {""};
+	// Leave empty to remove all. "Default" > leave original item.
+	faces[] = {"Default"};
+	// Leave empty to not change faces.
+	insignias[] = {};
+	// Leave empty to not change insignias
+
+	// All randomized
+	primaryWeapon[] = {"hlc_rifle_416D10"};
+	scope[] = {"CUP_optic_CompM2_low"};
+	bipod[] = {"CUP_bipod_Harris_1A2_L_BLK"};
+	attachment[] = {"CUP_acc_ANPEQ_15_Top_Flashlight_Black_L"};
+	silencer[] = {};
+	// Leave empty to remove all. "Default" for primaryWeapon > leave original weapon.
+
+	// Only *Weapons[] arrays are randomized
+	secondaryWeapon[] = {};
+	secondaryAttachments[] = {};
+	sidearmWeapon[] = {"CUP_hgun_Glock17_blk"};
+	sidearmAttachments[] = {"hlc_acc_DBALPL_FL"};
+	// Leave empty to remove all. "Default" for secondaryWeapon or sidearmWeapon > leave original weapon.
+
+	// These are added to the uniform or vest
+	magazines[] = {};
+	items[] = { MEDICAL_R ,"ACE_Flashlight_XL50",LIST_2("Chemlight_green")};
+	// These are added directly into their respective slots
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch"
+	};
+
+	// These are put into the backpack
+	backpackItems[] = {};
+
+	// Unit traits, see https://community.bistudio.com/wiki/setUnitTrait
+	traits[] = {};
+
+	// This is executed after unit init is complete. argument: _this = _unit.
+	code = "";
+
+class r : baseMan {
+	displayName = "Rifleman";
+	headgear[] = {"CUP_H_OpsCore_Spray"};
+	primaryWeapon[] = {"hlc_rifle_416D10"};
+	magazines[] = {
+		LIST_6("CUP_30Rnd_556x45_PMAG_QP"),
+		LIST_2("CUP_30Rnd_556x45_PMAG_QP_Tracer_Green"),
+		LIST_2("HandGrenade"),
+		LIST_2("SmokeShell")
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	primaryWeapon[] = {"hlc_rifle_416D10_gl"};
+	mamagazines[] = {
+		LIST_10("CUP_30Rnd_556x45_PMAG_QP"),
+		LIST_5("CUP_1Rnd_HE_M203"),
+		LIST_2("1Rnd_Smoke_Grenade_shell")
+		LIST_2("1Rnd_SmokeRed_Grenade_shell")
+		LIST_2("CUP_HandGrenade_M67")
+		LIST_2("SmokeShell")
+	};
+};
+class car : r {
+	displayName = "Carabinier";
+    primaryWeapon[] = {"hlc_rifle_416D10C"};
+	linkedItems[] += {"Binocular","ItemGPS","ItemCompass","ItemMap","ACE_MapTools"};
+
+};
+
+
+
+class cls : car {
+	displayName = "Combat Life Saver";
+	traits[] = {"medic"};
+	backpack[] = {"CUP_B_ACRPara_dpm"};
+	backpackItems[] = { MEDICAL_CLS };
+};
+
+class m : cls {
+	displayName = "Medic";
+	traits[] = {"medic"};
+	backpack[] = {"SP_Carryall_ATacsFG"};
+	backpackItems[] = { MEDICAL_M };
+};
+
+class smg : r {
+	displayName = "Submachinegunner";
+	primaryWeapon[] = {"CUP_smg_MP5A5"};
+	magazines[] = {
+		LIST_10("CUP_30Rnd_9x19_MP5"),
+		LIST_2("HandGrenade"),
+		LIST_2("SmokeShell")
+	};
+};
+
+class ftl : g {
+	displayName = "Fireteam Leader";
+	backpack[] = {"CUP_B_USPack_Coyote"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"Binocular","ItemGPS","ItemCompass"};
+};
+
+class sl : g {

```

Comment:
Inherit from ftl instead, so that you do not need to relist maptools, backpack, etc.